source/nodejs/adaptivecards-site/themes/adaptivecards/layout/_partial/header.ejs

```diff
@@ -51,43 +51,43 @@
                                 <nav id="uhf-g-nav" class="c-uhfh-gnav" role="presentation" data-m="{&quot;cN&quot;:&quot;Category nav_cont&quot;,&quot;cT&quot;:&quot;Container&quot;,&quot;id&quot;:&quot;c8c2m1r1a1&quot;,&quot;sN&quot;:8,&quot;aN&quot;:&quot;c2m1r1a1&quot;}">
                                     <ul class="js-paddle-items">
                                         <li class="single-link js-nav-menu x-hidden-none-mobile-vp">
-                                            <a class="c-uhf-nav-link" href="<%- url_for('/') %>" data-m="{&quot;cN&quot;:&quot;CatNav_Home_nav&quot;,&quot;id&quot;:&quot;n1c8c2m1r1a1&quot;,&quot;sN&quot;:1,&quot;aN&quot;:&quot;c8c2m1r1a1&quot;}">Home</a>
+                                            <a class="c-uhf-nav-link" href="<%- url_for('/') %>" data-m="{&quot;cN&quot;:&quot;CatNav_Home_nav&quot;,&quot;id&quot;:&quot;n1c8c2m1r1a1&quot;,&quot;sN&quot;:1,&quot;aN&quot;:&quot;c8c2m1r1a1&quot;}"><%= site.data.banner.en.home %></a>
```

Comment:
I suppose the same question applies here, just to be safe - this hardcoding of `en` will be replaced later when we have localized strings...is that correct?

Reply:
that's correct -- as noted in the description, there's still a little work left to do per-locale generation/detection. I didn't want to ship that on the site publically, because it'd be weird if we had e.g. https://adaptivecards.io/en-us/explorer, but no https://adaptivecards.io/es-es/explorer